### PR TITLE
Gv jbw diss uk

### PR DESCRIPTION
**Pull request by GV-JBW**
Added places to eat in Diss, Norfolk

### CODE REVIEW

File: GV-JBW-Diss-UK.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Diss, Norfolk, UK
+
+## Places to Eat
+1. Amandines for veggie food
+1. Pizza Time for :pizza:
+1. Other places
+1. I don't really know
+1. I don't live here just work
```
Comment (githubteacher) [inline GV-JBW-Diss-UK.md]: ❤️ the honesty 😄

File: GV-JBW-Diss-UK.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Diss, Norfolk, UK
+
+## Places to Eat
+1. Amandines for veggie food
+1. Pizza Time for :pizza:
```
Comment (githubteacher) [inline GV-JBW-Diss-UK.md]: what is your favorite pizza from Pizza time?

**Review by githubteacher — CHANGES_REQUESTED** [review]

File: GV-JBW-Diss-UK.md (hunk 1 above)
Comment (uppuravi) [inline GV-JBW-Diss-UK.md]: cool

File: GV-JBW-Diss-UK.md (hunk 2 above)
Comment (GV-JBW) [reply GV-JBW-Diss-UK.md]: anything with cheese dude !

**Review by uppuravi — CHANGES_REQUESTED** [review]
help

File: GV-JBW-Diss-UK.md (hunk 2 above)
Comment (aisruss) [reply GV-JBW-Diss-UK.md]: Their BBQ Chicken and Onion rings are delish! 28" pizzas from pizza time is better!

**Review by aisruss — APPROVED** [review]
Highly approve. even on the basis of pizza!

**Pull request merged**